src/Common/Thermodynamics/relations.jl

```diff
@@ -2074,3 +2075,11 @@ function moist_static_energy(
 ) where {FT <: Real}
     return specific_enthalpy(ts) + e_pot
 end
+
+"""
+    saturated(ts::ThermodynamicState)
+
+Boolean indicating if thermodynamic
+state is saturated.
+"""
+saturated(ts::ThermodynamicState) = (relative_humidity(ts) ≈ 1)
```

Comment:
@charleskawczynski - I think this assumes that you are running in equilibrium moisture setting. Wouldn't it be better to have `relative_humidity(ts) >= 1). This would include supersaturated conditions as well

Reply:
The [situation](https://github.com/CliMA/ClimateMachine.jl/pull/1308/files#diff-22c202328e500e0b200455bb04b99648R53) that @yairchn tested for was `relative_humidity(ts) ≈ 1`. @yairchn should we change this?

Reply:
I think Anna's suggestion is correct - it should be larger or equal 1

Reply:
Updated.